.github/workflows/test_and_deploy.yml

```diff
@@ -23,7 +22,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9, '3.10']
```

Comment:
I can undo this if needed (although I don't see why we shouldn't increment this)

Reply:
Ha! I just saw https://github.com/jni/zarpaint/pull/37
We must have been typing at the same time

Reply:
Ha! I just saw https://github.com/jni/zarpaint/pull/37
We must have been typing at the same time